Print_A_Bot/djwifi/templates/djwifi/configure.html

```diff
@@ -0,0 +1,246 @@
+                    <option value="wpa">{% trans 'WPA 1/2 (Hex [0-9/A-F])' %}</option>
+                    <option value="wpa-psk"{% verbatim %}{{#if_equals encryption 'WPA' }} selected{{/if_equals }}{{#if_equals encryption 'WPA2' }} selected{{/if_equals }}{% endverbatim %}>{% trans 'WPA 1/2 (Passphrase)' %}</option>
+                    <option data-divider="true"></option>
+                    <option value="wep-hex">{% trans 'WEP (Hex [0-9/A-F])' %}</option>
+                    <option value="wep-passphrase"{% verbatim %}{{#if_equals encryption 'WEP' }} selected{{/if_equals }}{% endverbatim %}>{% trans 'WEP (Passphrase)' %}</option>
+                </select>
+
+                <input type="password" id="passkey" class="form-control" name="wifi_password" placeholder="Password">
+                <p class="help-block">{% trans 'Passphrase or hex key depending on the encryption type selected.' %}</p>
+                <input id="show-password" type="checkbox"/> {% trans 'Show password' %}
+            </div>
+        </div>
+        <div class="modal-footer text-center">
+            <button type="button" class="btn btn-default" data-dismiss="modal">{% trans 'Close' %}</button>
+            <button type="submit" class="btn btn-primary">{% trans 'Save changes' %}</button>
+        </div>
+        <input type="hidden" name="wifi_ssid" value="{% verbatim %}{{ essid }}{% endverbatim %}">
+        {% csrf_token %}
+    </form>
+</script>
+
+<script id="template-wifi-submit" type="text/x-handlebars-template">
+    <div id="wifi-submit-wrapper" class="jumbotron text-center">
+        <h2>{% trans 'Garden Genie is Attempting to connect to' %} {% verbatim %}{{ essid }}{% endverbatim %}.</h2>
+
+        <p>
+            {% verbatim %}{{> waiting }}{% endverbatim %}
+        </p>
+
+        <p>{% trans 'When the light on your garden genie turns green you can check in through the cloud.' %}</p>
+        <p>{% trans 'Please re-connect to your regular wifi network and feel free to check back in with the Local button below.' %}</p>
+
+        <div class="row">
+            <div class="col-xs-6 col-sm-3 col-md-1 col-sm-offset-3 col-md-offset-5">
+                <a class="btn btn-success" href="http://gardengenie.local">{% trans 'Local' %}</a></div>
+            <div class="col-xs-6 col-sm-3 col-md-1">
+                <a class="btn btn-success" href="http://www.solargreens.net">{% trans 'Cloud' %}</a></div>
+        </div>
+    </div>
+</script>
+
+<script>
+    window.access_points = null;
+    window.access_point = null;
+
+    function configure_wifi(network_id) {
+        window.access_point = window.access_points[network_id];
+
+        console.log('access_point');
+        console.log(window.access_point);
+        if (window.access_point.encrypted) {
+            var template = Handlebars.compile($('#template-wifi-configure').html());
+            show_modal_content(template(window.access_point));
```

Comment:
window.access point is used as a global place to store variables similar to a dict